src/test/scala/org/ergoplatform/http/routes/ScriptApiRouteSpec.scala

```diff
@@ -56,7 +56,7 @@ class ScriptApiRouteSpec extends AnyFlatSpec
       val cost = json.hcursor.downField("cost").as[Int].right.get
       value shouldEqual -45
       condition shouldEqual true
-      cost shouldEqual 40
+      cost shouldEqual 50
```

Comment:
This new cost is correct, because this is how the tree is actually validated as part of transaction.